### ISSUE TITLE
plot_path(model, path, X_test, y_test)

### ISSUE THREAD

**Issue by thiagoribeiro1** [lasso-net/lassonet#4]
Firstly, I would like to congratulate for the amazing solution you have developed. I am using it in a classification problem (ANN). I just can't figure out what are the features in the number of selected features (x-axis). Please, can you help me? Thank you very much!

**Comment by ilemhadri** [comment]
Glad to hear our software is useful!

> I just can't figure out what are the features in the number of selected features (x-axis).

The input_mask method does just that: https://github.com/lasso-net/lassonet/blob/master/lassonet/model.py#L45

**Comment by thiagoribeiro1** [comment]
Thank you very much for your explanation, @ilemhadri! Sorry for my ignorance, but how could I have access to this method? Is there an example, please?

**Comment by louisabraham** [comment]
The returned path is a list of
```py
@dataclass
class HistoryItem:
    lambda_: float
    state_dict: dict
    objective: float  # loss + lambda_ * regulatization
    loss: float
    val_objective: float  # val_loss + lambda_ * regulatization
    val_loss: float
    regularization: float
    selected: torch.BoolTensor
    n_iters: int
```

So you can do `path[i].selected` and you will get a BoolTensor indicating for each feature whether it was selected.
